hivemind/client/averaging/training.py

```diff
@@ -47,55 +47,63 @@ def __init__(self, opt: torch.optim.Optimizer, *, average_parameters: bool, aver
             averaged_tensors = [tensor.detach().cpu().float().clone() for tensor in self.local_tensors()]
         super().__init__(averaged_tensors=averaged_tensors, **kwargs)
 
-    @torch.no_grad()
```

Comment:
Why removed: because decorators are incompatible with our current RTFD parser

(this unlocks #288 )